process-engine/process_engine_runtime
# Add Autostart for Windows

## What did you change?

* added `.bat`-scripts for autostart on Windows
* updated documentation of autostart in `README.md`

## How can others test the changes?

On a windows machine:
* `npm i -g @process-engine/process_engine_runtime@feature~add_autostart_for_windows`
* Right-click the `start_runtime_after_system_boot.bat`-script and choose `Run as Administrator` (please see the `README.md` of this feature branch for detailed instructions)
* Restart the windows machine and log into your user account
* The `process_engine_runtime` should be running
   * The easiest test is to open a browser and visit `localhost:8000`
      * Result will be `Cannot GET /`
   * Additionaly you can run `pm2 list` in the terminal
      * You should see one entry with status `online`

## PR-Checklist

Please check the boxes in this list after submitting your PR:
